src/lib/tusktask/invalidators/notifications/invalidators.tsx

```
@@ -1,44 +1,93 @@
+  const setOpenIndex = useChatStore((s) => s.setOpenIndex);
+
+  const invalidateByNotificationType = (notification: NotificationType) => {
+    switch (notification.type) {
       case "newRoomChat":
         queryClient.invalidateQueries({
           queryKey: ["conversations"],
         });
         break;
+
       case "directMessage":
+        const conversationId = notification?.payload?.conversationId;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)